Sample/Droid/Properties/AndroidManifest.xml

```diff
@@ -1,9 +1,11 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android" android:versionCode="1" android:versionName="1.0" package="com.example.download_example">
-	<uses-sdk android:minSdkVersion="15" android:targetSdkVersion="25" />
+	<uses-sdk android:minSdkVersion="15" android:targetSdkVersion="28" />
 	<!--
 	You have to add this permission when setting `PathNameForDownloadedFile`.
 	<uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
 	-->
-	<application android:label="Download Example"></application>
+  
+  <!-- usesCleartextTraffic must be enabled for Android 9 and downloads using http -->
+	<application android:label="Download Example" android:usesCleartextTraffic="true"></application>
```

Comment:
Is this only needed for Android 9 when trying to download content via HTTP, or does it also apply for older Versions of Android - or other protocols?

Reply:
Below Android 9 the defaults to android:usesCleartextTraffic="true".
Android 9 is the first project that sets android:usesCleartextTraffic="false" as default.
A downloadmanager that can't handle http downloads isn't very handy in my eye. But using http is considered as a safety risk.